src/file_tree.ml

```diff
@@ -194,9 +198,7 @@ let load ?(extra_ignored_subtrees=Path.Set.empty) path =
               | _ ->
                 (false, File.dummy)
             in
-            if ignore_file fn ~is_directory then
```

Comment:
This used to skip files of the form `.#XXX`, i.e. emacs temporary files, so we still need it. Though we can simplify it a bit.

Reply:
Ah yeah, thats' a good point. I brought this back and added a similar exception for vim files.